src/components/UpcomingEventsList.tsx

```diff
@@ -5,24 +5,14 @@ import Translation from "@/components/Translation"
 
 import { trackCustomEvent } from "@/lib/utils/matomo"
 
-import events from "../data/community-events.json"
+import CommunityEvents, { CommunityEvent } from "../data/community-events"
 
 import { Button } from "./Buttons"
 import EventCard from "./EventCard"
 import InfoBanner from "./InfoBanner"
 import InlineLink from "./Link"
 
-interface ICommunityEventData {
-  title: string
-  to: string
-  sponsor: string | null
-  location: string
-  description: string
-  startDate: string
-  endDate: string
-}
-
-interface IOrderedUpcomingEventType extends ICommunityEventData {
+interface IOrderedUpcomingEventType extends CommunityEvent {
```

Comment:
Would refactor to `type`, ditch the `I` prefix and `Type` suffix in the name, and consider moving to `@/lib/types` (though for this one it doesn't seem as imperative, if you want to just keep it here).